gitlab/tests/conftest.py

```diff
@@ -4,14 +4,16 @@
 import copy
 import json
 import os
+from contextlib import contextmanager
 from time import sleep
 
 import mock
 import pytest
 import requests
 from six import PY2
 
-from datadog_checks.dev import docker_run
+from datadog_checks.dev import EnvVars, TempDir, docker_run
+from datadog_checks.dev._env import get_state, save_state
```

Comment:
These are always exposed as fixtures if you want to do that instead https://github.com/DataDog/integrations-core/blob/171dfc38782fd51eff4bdaa88daaf083bf04659c/datadog_checks_dev/datadog_checks/dev/plugin/pytest.py#L241-L248

Reply:
I did not know that thanks Ofek! I'll keep it that way for this PR